tutorials/ph-ug-temp/index.md

```diff
@@ -0,0 +1,102 @@
+
+1. Pull in a shape file of Uganda. 
+
+    In these lines of code, we are creating a new variable called region as before. However, now we are pulling in a “FeatureCollection” object, and filtering by ‘Country’ to select ‘Uganda.’ FeatureCollections are groups of features (spatial data and attributes).  “Filter” is the command to extract a specific set of feature data from a feature collection. We then map it.
+
+  ```
+  var ug = ee.FeatureCollection('ft:1tdSwUL7MVpOauSgRzqVTOwdfy17KDbw-1d9omPw') 
+    .filter(ee.Filter.eq('Country', 'Uganda'));
+
+  Map.addLayer(ug)
+  ```
+2. Read in land surface data.
+
+    Now, we need to read in land surface temperature data. 'ImageCollection' objects store collections of images. Using the Earth Engine Data Catalog, we can find different image collections. The 'ImageCollection' we are reading in here contains the LST data, but we can read in a different 'ImageCollection' for other types of data (e.g. vegetation index, or rain fall, light at night).
+Our code applies filters to restrict images for Uganda, within January to December 2015.
+
+```
+
+var modis = ee.ImageCollection('MODIS/MOD11A2') // read in image collection
+
+var mod11a2 = modis.
+  filterBounds(ug). // just pull for Uganda
+  filterDate('2015-01-01', '2015-12-31'); // take 1 year (2015)
+
+var modLSTday = mod11a2.select('LST_Day_1km'); // pull just 1km day LST
+
+```
+3. Create a function to convert temperature units.
+
+    We then need to convert temperature from Kelvin to Celsius. We can do this by writing a function to do the conversion, and then by mapping over the data.
+
+```
+var modLSTc = modLSTday.map(function(img) {
+  return img.multiply(0.02).subtract(273.15).copyProperties(img,['system:time_start','system:time_end']); 
+}); 
+
+```
+4. Describe land surface temperature data using time series.
+
+    It can be helpful to describe your data using a time series graph. We can plot the mean land surface temperature over the year using the following code:
+
+```
+// Charts Long-term Time Series
+var TS1 = ui.Chart.image.series(modLSTc, ug,
+ ee.Reducer.mean(), 1000, 'system:time_start').setOptions({
+   title: 'LST Long-Term Time Series',
+   vAxis: {title: 'LST Celsius'},
+ });
+ print(TS1);
+
+```
+
+5. Visualize processed data on a map.
+
+    Now, we may want to visualize our data. We can take the mean LST in celsius, and clip to Uganda. This code will produce a map of the mean temperatures for us to view in the map window.
+
+```
+
+var clippedLSTc = modLSTc.mean().clip(ug);
+// Add clipped image layer to map
+Map.addLayer(clippedLSTc, {'min': 0, 'max': 40, 'palette':"0000ff,32cd32,ffff00,ff8c00,ff0000 "});
+
+```
+6. Export data for further analysis.
+
+    Finally, we need to export our raster image file to perform further analysis (e.g. link to participant or clinic data). We can use the export command to download the processed image data to our Google Drive folder.
```

Comment:
Please explain the args to `toDrive`.  Where can the user find the result?  What will it be called?

Reply:
I have added these details